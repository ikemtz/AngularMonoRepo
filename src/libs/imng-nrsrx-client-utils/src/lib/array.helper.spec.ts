import { flat } from './array.helper';
describe('array.helper', () => {
  it('flat should work 2 deep', () => {
    const array = [1, 2, 3, [4, 5, [6]]];
    expect(flat(array)).toMatchSnapshot();
  });

  it('flat should work 3 deep', () => {
    const array = [1, 2, 3, [4, 5, [6]]];
    expect(flat(flat(array))).toMatchSnapshot();
  });
});
