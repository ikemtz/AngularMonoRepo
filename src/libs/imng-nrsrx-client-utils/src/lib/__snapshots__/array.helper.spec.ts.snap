// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`array.helper flat should work 2 deep 1`] = `
[
  1,
  2,
  3,
  4,
  5,
  [
    6,
  ],
]
`;

exports[`array.helper flat should work 3 deep 1`] = `
[
  1,
  2,
  3,
  4,
  5,
  6,
]
`;
